packages/wallet-sdk/src/provider/CoinbaseWalletProvider.test.ts

```diff
@@ -740,10 +763,7 @@ describe("CoinbaseWalletProvider", () => {
 
       jest.spyOn(relay, "switchEthereumChain").mockReturnValue({
         cancel: () => {},
-        promise: Promise.resolve({
-          method: Web3Method.switchEthereumChain,
-          errorCode: 4092,
```

Comment:
the correct error code for switch chain error is actually `4902` not `4092`.
we've been providing wrong information. 😭 
https://www.google.com/search?q=4902+wallet_switchEthereumChain
![image](https://user-images.githubusercontent.com/966662/227610182-3b0e396d-3ffa-4595-a7b6-2a247e054b2c.png)